utils/prisma/prismaHelpers.ts

```diff
@@ -0,0 +1,47 @@
+import { Prisma } from '@prisma/client';
+import { NextApiResponse } from 'next';
+import { StatusMessageType } from '../../types/apiResponse';
+import { createJsonResponse, HTTP_STATUS_BAD_REQUEST, HTTP_STATUS_INTERNAL_SERVER_ERROR } from '../http/httpHelper';
+
+/**
+ * Utility function that takes care of Prisma runtime errors during CREATE.
+ *
+ * @param res The NextApiResponse object.
+ * @param creator The Prisma Creator function.
+ */
+export function createIfPossible(res: NextApiResponse, creator: () => Promise<void>) {
```

Comment:
```suggestion
export function handlePrismaAction(res: NextApiResponse, prismaAction: () => Promise<void>) {
```
Is this function specific to `CREATE` actions? If it is then just ignore this comment.